stdlib/Markdown/src/render/terminal/render.jl

```diff
@@ -156,20 +183,31 @@ function terminline(io::IO, md::Image)
     terminline(io, "(Image: $(md.alt))")
 end
 
-terminline(io::IO, f::Footnote) = with_output_color(terminline, :bold, io, "[^$(f.id)]")
+function terminline(io::IO, f::Footnote)
+    print(io, styled"{markdown_footnote:[^$(f.id)]}")
+end
 
-function terminline(io::IO, md::Link)
-    url = !Base.startswith(md.url, "@ref") ? " ($(md.url))" : ""
-    text = terminline_string(io, md.text)
-    terminline(io, text, url)
+function terminline(io::AnnotIO, md::Link)
+    annots = if !isnothing(match(r"^(?:https?://|file://)", md.url))
+        (:face => :markdown_link, :link => md.url)
+    else
+        (:face => :markdown_link,)
+    end
+    with_output_annotations(io -> terminline(io, md.text), io, annots...)
 end
 
 function terminline(io::IO, code::Code)
-    printstyled(io, code.code, color=:cyan)
+    print(io, styled"{markdown_inlinecode:$(code.code)}")
 end
 
 function terminline(io::IO, tex::LaTeX)
-    printstyled(io, tex.formula, color=:magenta)
+    print(io, styled"{markdown_latex:$(tex.formula)}")
+end
+
+function terminline(io::IO, md::MarkdownElement)
+    a = IOContext(AnnotatedIOBuffer(), io)
+    terminline(a, md)
+    print(io, read(a.io, AnnotatedString))
```

Comment:
ibid

Reply:
ibid?

Reply:
Sorry, mixing up my Latin terms. I guess this should use _idem_, meaning 'the same' to mark that the same comment as above applies here